### PR TITLE
[DEX][Pair] Market Active State

### PR DESCRIPTION
**Pull request by crishellco**
## Ticket
* n/a

## Changes
* Add active class and styles
* Lint

## Screenshots
image

### CODE REVIEW

File: src/components/dex/OrdersTabs.vue [hunk 1]
```diff
@@ -9,41 +9,24 @@
...
     return {
       tabOptions: [
-        {key: 'my-orders', label: 'My orders'}
-      ,
-        {key: 'order-book', label: 'Order book'}
-      ,
-        {key: 'trade-history', label: 'Trade history'}
+        { key: 'my-orders', label: 'My orders' },
+        { key: 'order-book', label: 'Order book' },
+        { key: 'trade-history', label: 'Trade history' },
```
Comment (schaeferCode) [inline src/components/dex/OrdersTabs.vue:19]: Yep, way better. HA

File: src/components/dex/Pair.vue [hunk 2]
```diff
@@ -183,6 +191,10 @@ export default {
           color: $red;
         }
       }
+
+      .row.active {
+        background: lighten($dark-purple, 10%);
+      }
```
Comment (schaeferCode) [inline src/components/dex/Pair.vue:201]: I have some `.active` row styling in the simpleTable component already so you'll probably wanna remove that. This is def a more flexible solution.

**Review by schaeferCode — APPROVED** [review]
Looks great! One more thing though, I think it would look better if the borders disappeared when the row became active...unless I'm wrong.

File: src/components/dex/OrdersTable.vue [hunk 3]
```diff
@@ -55,7 +55,32 @@ export default {
...
     ordersTableData() {
-      return [{pairAndSide: {pair: 'APH/NEO', side: 'buy'}, details: {amount: '72,043.56', base: 'APH', price: '0.225', cost: '$.25'}}, {pairAndSide: {pair: 'APH/ATI', side: 'sell'}, details: {amount: '72,043.56', base: 'APH', price: '0.225', cost: '$.25'}}];
+      return [
+        {
+          pairAndSide: {
+            pair: 'APH/NEO',
+            side: 'buy',
+          },
+          details: {
+            amount: '72,043.56',
```
Comment (jsolman) [inline src/components/dex/OrdersTable.vue:67]: why is this mock data formatted as a string with a comma?

File: src/components/dex/OrdersTable.vue [hunk 4]
```diff
@@ -55,7 +55,32 @@ export default {
...
+            base: 'APH',
+            price: '0.225',
```
Comment (jsolman) [inline src/components/dex/OrdersTable.vue:69]: why string?

File: src/components/dex/OrdersTable.vue [hunk 5]
```diff
@@ -55,7 +55,32 @@ export default {
...
+            cost: '$.25',
```
Comment (jsolman) [inline src/components/dex/OrdersTable.vue:70]: what cost is this, why is it in the mock input data for an order?

**Review by jsolman — APPROVED** [review]
changes look good -- my questions related to mock order data aren't really related to other content in this PR.

File: src/components/dex/OrdersTable.vue (hunk 5 above)
Comment (crishellco) [reply src/components/dex/OrdersTable.vue:70]: I just formatted what was there. @schaeferCode?

Comment (schaeferCode) [reply src/components/dex/OrdersTable.vue:70]: I just made up this data based on what I saw in the design. No clue if this data makes sense or not, but it seemed appropriate enough for our designer to throw it into the design, so I figured it was appropriate enough to add as fake data.

**Pull request merged**